src/components/Epic.js

```diff
@@ -0,0 +1,27 @@
+// @flow
```

Comment:
I think this component should be called `MembershipEngagementBanner` or similar. The `Epic` terminology relates to this "Since you're here" box embedded in articles:

![picture 529](https://user-images.githubusercontent.com/5931528/37825999-c3c2091a-2e89-11e8-9271-e8b7e01ca385.png)